### PR TITLE
Fixing E2E test failures due to missing env variables.

### PR DESCRIPTION
**Pull request by v-iyamauchi**
## Description
Fixing E2E test failures on local runs due to missing env variables.

## Related issues
Addresses [issue #126953].

[Bug 126953](https://microsofthealth.visualstudio.com/Health/_workitems/edit/126953): GivenImportInvalidResource_ThenErrorLogsShouldBeOutput test fails locally

## Testing
Describe how this change was tested.

## FHIR Team Checklist
- **Update the title** of the PR to be succinct and less than 65 characters
- **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- Tag the PR with the type of update: **Bug**, **Build**, **Dependencies**, **Enhancement**, **New-Feature** or **Documentation**
- Tag the PR with **Open source**, **Azure API for FHIR** (CosmosDB or common code) or **Azure Healthcare APIs** (SQL or common code) to specify where this change is intended to be released.
- Tag the PR with **Schema Version backward compatible** or **Schema Version backward incompatible** or **Schema Version unchanged** if this adds or updates Sql script which is/is not backward compatible with the code.
- [ ] CI is green before merge [Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main) 
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj [hunk 1]
```diff
@@ -302,5 +302,6 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Health.Fhir.Api\Microsoft.Health.Fhir.Api.csproj" />
     <ProjectReference Include="..\Microsoft.Health.Fhir.Core\Microsoft.Health.Fhir.Core.csproj" />
+    <ProjectReference Include="..\Microsoft.Health.Fhir.CosmosDb\Microsoft.Health.Fhir.CosmosDb.csproj" />
```
Comment (brendankowitz) [inline src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj:305]: Whats the dependency here?

Comment (v-iyamauchi) [reply src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj:305]: That project has two constants defined for the storage emulator host name and key. I thought about defining them locally in EnvironmentVariables class, but wanted to check how everyone thinks about adding this dependency. Please let me know what you think.

File: test/Microsoft.Health.Fhir.Shared.Tests.Crucible/CrucibleDataSource.cs [hunk 2]
```diff
@@ -24,11 +25,11 @@ public CrucibleDataSource(DataStore dataStore)
             TestRun = new Lazy<Task>(TestRunAsync);
         }
 
-        public static string CrucibleEnvironmentUrl => Environment.GetEnvironmentVariable("CrucibleEnvironmentUrl");
+        public static string CrucibleEnvironmentUrl => EnvironmentVariables.GetEnvironmentVariable(KnownEnvironmentVariableNames.CrucibleEnvironmentUrl);
```
[the remark this answers is not in the document]
Comment (v-iyamauchi) [reply test/Microsoft.Health.Fhir.Shared.Tests.Crucible/CrucibleDataSource.cs:28]: Okay, will remove it in a next iteration.

File: src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj (hunk 1 above)
Comment (mikaelweave) [reply src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj:305]: I'd prefer not adding the dependency and copying the values over locally. I think in general common packages shouldn't rely on specific packages (like for datastore). It should be the other way around.

Comment (v-iyamauchi) [reply src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj]: Fixed as suggested.

**Review by mikaelweave — APPROVED** [review]

**Pull request merged**